@navikt/ds-tokens/src/index.js

```diff
@@ -1,24 +1,39 @@
 const Color = require("color");
 
+// https://github.com/hihayk/scale/blob/69b766bba2db046d3e8cb4026ae32a32c897f9ff/src/utils.js#L44
+const mixColors = (color, step, amount, mixColor) => {
+  const saturation = Math.round(Color(color).hsl().color[1]);
+  // Setting directly to hex returns different result...
+  return Color(
+    Color(color)
+      .saturate(((step + 1) / 5) * (saturation / 100))
+      .mix(Color(mixColor), ((amount / 100) * (step + 1)) / 5)
+      .string()
+  ).hex();
+};
+
+const lightColor = (color, step) => mixColors(color, step, 100, "white");
+const darkColor = (color, step) => mixColors(color, step, 85, "black");
+
 const baseColors = {
   blue: "#0067c5",
   deepblue: "#005B82",
   lightblue: "#66CBEC",
   orange: "#FF9100",
   green: "#06893A",
   red: "#BA3A26",
+  purple: "#634689",
+  limegreen: "#a2ad00",
 };
+
 const white = "#ffffff";
-const darkgray = "#3e3832";
+const darkgray = "#262626";
 
-const mix = (a, b, percentage) =>
-  Color(a)
-    .mix(Color(b), percentage / 100)
+const lighten = (color, percentage) =>
```

Comment:
Denne kan vel slettes nå?

Reply:
Blir nå brukt for hover-farge i hoved/fare-knapp, men tar å inliner den.